bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/HomeCapability.java

```diff
@@ -58,7 +58,13 @@ public HomeCapability(CommonInterface handler, NetatmoDescriptionProvider descri
     public void initialize() {
         super.initialize();
         HomeConfiguration config = handler.getConfiguration().as(HomeConfiguration.class);
-        homeIds = Set.of(config.getId(), config.energyId, config.securityId);
+        homeIds.add(config.getId());
```

Comment:
Should the set be cleared when `dispose` is called? Otherwise configuration changes which result in `dispose`/`initialize` will lead to the "old" values still be present in the set.

Reply:
You're correct on this.